### PR TITLE
JENKINS-20319 enhancement

### PR DESCRIPTION
**Pull request by iaon**
Placeholder (send_to_individuals_who_broke) has been added to handle "Send separate e-mails to individuals who broke the build"
- For CoreEmailSlicer placeholder feature is fully functional. We can view, set and remove such type of notification.
- For ExtEmailSlicer placeholder works in read only. Set or remove doesn't make affect.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -92,32 +90,30 @@
...
-        <url>https://github.com/jenkinsci/configurationslicing-plugin</url>
-      <tag>HEAD</tag>
-  </scm>
-
+	<scm>
+		<connection>scm:git:git@github.com:jenkinsci/configurationslicing-plugin.git</connection>
+		<developerConnection>scm:git:git@github.com:jenkinsci/configurationslicing-plugin.git</developerConnection>
+		<url>https://github.com/jenkinsci/configurationslicing-plugin</url>
+		<tag>HEAD</tag>
+	</scm>
```
Comment (batmat) [inline pom.xml]: Hi!
Those changes above IMO shouldn't be there. All seem to be whitespace/formatting changes, unrelated to the goal of that PR.

File: pom.xml [hunk 2]
```diff
@@ -128,28 +124,22 @@
...
+			<url>http://repo.jenkins-ci.org/public/</url>
+		</repository>
+	</repositories>
+	<pluginRepositories>
+		<pluginRepository>
+			<id>repo.jenkins-ci.org</id>
+			<url>http://repo.jenkins-ci.org/public/</url>
+		</pluginRepository>
+	</pluginRepositories>
+</project>
```
Comment (batmat) [inline pom.xml]: Same here, shouldn't have been committed.

**Comment by guysoft** [comment]
Hey, adopted this plugin, this PR conflicts now, any change you can fix it so I can merge it? I am not an expert on Java and I don't want to break anything.
There is a new version coming out now that I got this compiling. So it would be geat if you could.

**Comment by guysoft** [comment]
Closing if no reply

**Comment by guysoft** [comment]
Hey @iaon any news?
Don't mind checking up if that helps you come up with a new PR

**Comment by MarkEWaite** [comment]
Closing pull request after no response

**Pull request closed without merging**